docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -1345,444 +1370,119 @@ To create, accept, or cancel escrows, it's better to implement mutations for eac
+The UI has a tab for users to manage their owned objects to be ready for escrow trading. The code of this tab lives in the file `src/routes/LockedDashBoard.tsx`.
+
+In this tab, there are two sub-tabs:
 
 - **My Locked Objects** tab to list out all of owned `Locked` objects.
 - **Lock Own Objects** tab to lock owned objects.
 
+{@inject: examples/trading/frontend/src/routes/LockedDashboard.tsx}
+
 #### My Locked Objects tab
 
 Let's take a look at the **My Locked Objects** tab by examining `src/components/locked/OwnedLockedList.tsx`. Focus on the logic on how to retrieve this list.
 
-```ts title='src/components/locked/OwnedLockedList.tsx'
-import { useCurrentAccount, useSuiClientInfiniteQuery } from '@mysten/dapp-kit';
-
-import { InfiniteScrollArea } from '@/components/InfiniteScrollArea';
-import { CONSTANTS } from '@/constants';
-
-import { LockedObject } from './LockedObject';
-
-/**
- * Similar to the `ApiLockedList` but fetches the owned locked objects
- * but fetches the objects from the on-chain state, instead of relying on the indexer API.
- */
-export function OwnedLockedList() {
-	const account = useCurrentAccount();
-	const { data, isLoading, fetchNextPage, hasNextPage, isFetchingNextPage } =
-		useSuiClientInfiniteQuery(
-			'getOwnedObjects',
-			{
-				filter: {
-					StructType: CONSTANTS.escrowContract.lockedType,
-				},
-				owner: account?.address!,
-				options: {
-					showContent: true,
-					showOwner: true,
-				},
-			},
-			{
-				enabled: !!account?.address,
-				select: (data) => data.pages.flatMap((page) => page.data),
-			},
-		);
-	return (
-		<>
-			<InfiniteScrollArea
-				loadMore={() => fetchNextPage()}
-				hasNextPage={hasNextPage}
-				loading={isFetchingNextPage || isLoading}
-			>
-				{data?.map((item) => <LockedObject key={item.data?.objectId} object={item.data!} />)}
-			</InfiniteScrollArea>
-		</>
-	);
-}
-```
+{@inject: examples/trading/frontend/src/components/locked/OwnedLockedList.tsx}
 
 Fetch the owned `Locked` objects directly from Sui blockchain using the `useSuiClientInfiniteQuery()` hook from `dApp Kit`. This hook is a thin wrapper around Sui blockchain RPC calls, reference the documentation to learn more about these [RPC hooks](https://sdk.mystenlabs.com/dapp-kit/rpc-hooks). Basically, supply the RPC endpoint you want to execute, in this case it's the [`getOwnedObjects` endpoint](https://docs.sui.io/sui-api-ref#suix_getownedobjects). Supply the connected wallet account and the `Locked` object struct type to the call. The struct type is usually identified by the format of `{PACKAGE_ID}::{{MODULE_NAME}}::{{STRUCT_TYPE}}`. The returned data is stored inside the cache at query key `getOwnedObjects`. Recall the previous section where you invalidate the data at this key after the mutation succeeds, the `useSuiClientInfiniteQuery()` hook automatically re-fetches the data, thus you don't have to worry about the out-dated data living in your frontend application.
```

Comment:
nit: Use of the term "blockchain" stood out here, because I don't see it often in our docs, but I'll defer to @ronny-mysten to do a terminology "sweep".

Reply:
I'll defer to @ronny-mysten